templates/launch.html

```diff
@@ -373,14 +373,15 @@ <h3>Runner Data</h3>
                         document.querySelector("#survey_metadata").innerHTML = schema_response.metadata.map(metadataField => {
 
                             const fieldName = metadataField["name"]
-                            let defaultValue = metadataField['default'];
 
                             if (metadataField['type'] === "date") {
                                 const currentDate = new Date()
                                 // format as yyyy-MM-dd
                                 defaultValue = currentDate.toLocaleDateString().split("/").reverse().join("-")
                             }
 
+                            defaultValue = metadataField['default'];
```

Comment:
ah I actually deleted this line in my PR with Kyle as I thought it was an accident, if we don't want default date I think the
```
if (metadataField['type'] === "date") {
   const currentDate = new Date()
   // format as yyyy-MM-dd
   defaultValue = currentDate.toLocaleDateString().split("/").reverse().join("-")
}
```
 can be removed to avoid confusion